test_timeout.py

```diff
@@ -6,6 +6,7 @@
 
 
 TIMEOUTS_MS = (2, 10, 50, 100, 200, 1000, 1500, 5000, 20000)
+NEGATIVE_TIMEOUTS_MS = (-100, -10, -1, 0)
```

Comment:
Zero is not a negative number.

Reply:
I know, but `JUST_SOME_OTHER_TIMEOUTS_WE_WANT_TO_TEST_MS` didn't seem quite right either and no better alternative crossed my mind yesterday. What about `POSITIVE_TIMEOUTS_MS` and `TIMEOUTS_MS` instead? That would be more accurate.